pkg/networkservice/chains/endpoint/server.go

```diff
@@ -53,14 +51,15 @@ type endpoint struct {
 // NewServer - returns a NetworkServiceMesh client as a chain of the standard Client pieces plus whatever
 //             additional functionality is specified
 //             - name - name of the NetworkServiceServer
+//             - authServer authorization server chain element
 //             - tokenGenerator - token.GeneratorFunc - generates tokens for use in Path
 //             - additionalFunctionality - any additional NetworkServiceServer chain elements to be included in the chain
-func NewServer(name string, authzPolicy *rego.PreparedEvalQuery, tokenGenerator token.GeneratorFunc, additionalFunctionality ...networkservice.NetworkServiceServer) Endpoint {
+func NewServer(name string, authServer networkservice.NetworkServiceServer, tokenGenerator token.GeneratorFunc, additionalFunctionality ...networkservice.NetworkServiceServer) Endpoint {
```

Comment:
please make this authz ... so as not to confuse it with authn :)

Reply:
Comment above says that it is authorization server :)
But ok, fixed.